app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,41 @@
 class PostsController < ApplicationController
-  def index; end
-  
-  def show; end
-end
\ No newline at end of file
+  def index
+    @current_user = current_user
+    @posts_per_page = 2
+    @user = User.find(params[:user_id])
+    @page = params.fetch(:page, 1)
+    # rawposts = @user.posts.includes(:comments)
+    # @posts = rawposts[2 * (@page.to_i - 1), @posts_per_page]
```

Comment:
[OPTIONAL] Please kindly delete unused code for it makes your code base look Untidy.